src/shared-components/button/components/textButton/style.ts

```diff
@@ -19,5 +18,6 @@ export const BaseTextButton = styled.button<{ disabled: boolean }>`
   &:active,
   &:focus {
     outline: none;
+    box-shadow: ${BOX_SHADOWS.focusSecondary};
```

Comment:
Just a note that I'm going to add a note to our pitch to look at potentially defining `FOCUS.secondary` - might be helpful to remove some of the duplication and make things a bit more expressive. Definitely not needed for now though.